lib/mongo.rb

```diff
@@ -75,8 +75,16 @@
 require 'mongo/version'
 require 'mongo/write_concern'
 require 'mongo/utils'
+require 'mongo/config'
 
 module Mongo
+  extend Forwardable
+  extend self
+
+  # Take all the public instance methods from the Config singleton and allow
+  # them to be accessed through the Mongo module directly.
+  def_delegators Config, *(Config.public_instance_methods(false))
```

Comment:
I would sleep better if this was an explicit enumeration of methods, how difficult would that be to implement?

Reply:
done